docs/tutorials/word-tutorial.md

```diff
@@ -0,0 +1,962 @@
+        <button class="ms-Button" id="insert-text-into-range">Insert Abbreviation</button>
+    </div>
+    ```
+
+4. Open the app.js file.
+
+5. Below the line that assigns a click handler to the `change-font` button, add the following code:
+
+    ```js
+    $('#insert-text-into-range').click(insertTextIntoRange);
+    ```
+
+6. Below the `changeFont` function, add the following function:
+
+    ```js
+    function insertTextIntoRange() {
+        Word.run(function (context) {
+
+            // TODO1: Queue commands to insert text into a selected range.
+
+            // TODO2: Load the text of the range and sync so that the
+            //        current range text can be read.
+
+            // TODO3: Queue commands to repeat the text of the original
+            //        range at the end of the document.
+
+            return context.sync();
+        })
+        .catch(function (error) {
+            console.log("Error: " + error);
+            if (error instanceof OfficeExtension.Error) {
+                console.log("Debug info: " + JSON.stringify(error.debugInfo));
+            }
+        });
+    }
+    ``` 
+
+7. Replace `TODO1` with the following code. Note:
+
+   - The method is intended to insert the abbreviation ["(C2R)"] into the end of the Range whose text is "Click-to-Run". It makes a simplifying assumption that the string is present and the user has selected it.
+
+   - The first parameter of the `Range.insertText` method is the string to insert into the `Range` object.
+
+   - The second parameter specifies where in the range the additional text should be inserted. Besides "End", the other possible options are "Start", "Before", "After", and "Replace". 
+
+   - The difference between "End" and "After" is that "End" inserts the new text inside the end of the existing range, but "After" creates a new range with the string and inserts the new range after the existing range. Similarly, "Start" inserts text inside the beginning of the existing range and "Before" inserts a new range. "Replace" replaces the text of the existing range with the string in the first parameter.
+
+   - You saw in an earlier stage of the tutorial that the insert* methods of the body object do not have the "Before" and "After" options. This is because you can't put content outside of the document's body.
+
+    ```js
+    const doc = context.document;
+    const originalRange = doc.getSelection();
+    originalRange.insertText(" (C2R)", "End");
+    ```
+
+8. We'll skip over `TODO2` until the next section. Replace `TODO3` with the following code. This code is similar to the code you created in the first stage of the tutorial, except that now you are inserting a new paragraph at the end of the document instead of at the start. This new paragraph will demonstrate that the new text is now part of the original range.
+
+    ```js
+    doc.body.insertParagraph("Original range: " + originalRange.text,
+                             "End");
```

Comment:
Nit: Could we get rid of this line break?

Reply:
Done.